src/test/java/com/lpvs/controller/LPVSWebControllerTest.java

```diff
@@ -66,7 +66,7 @@ public void testPersonalInfoSettings() {
         when(loginCheckService.getMemberFromMemberMap(authentication)).thenReturn(member);
 
         LPVSMember result =
-                webController.new PublicInterface().personalInfoSettings(authentication);
+                webController.new WebApiEndpoints().personalInfoSettings(authentication);
```

Comment:
Same question with this test file.

Reply:
After updating the name of an interface, I have to update UTs as well.